initrd-progs/0initrd/init

```diff
@@ -785,7 +785,7 @@ remount_overlay() {
   umount -l /pup_new
  else
   mkdir /mnt/tmpfs/pup_work
-  OPTS="lowerdir=,upperdir=/mnt/tmpfs/pup_rw,workdir=/mnt/tmpfs/pup_work,xino=on,index=on"
+  OPTS="lowerdir=,upperdir=/mnt/tmpfs/pup_rw,workdir=/mnt/tmpfs/pup_work,xino=on"
```

Comment:
In current kernels  both OVERLAY_FS_INDEX and OVERLAY_FS_XINO_AUTO are not set (also OVERLAY_FS_INDEX),
but are good features to have. Better activate them in the kernel rather than remove them

Reply:
OVERLAY_FS_INDEX and OVERLAY_FS_XINO_AUTO determine whether or not to auto-enable these features for a newly mounted overlay. Support for them is always included, and these features are active if the build-time option is enabled, or if enabled explicitly via mount options.

Reply:
Even when kernel config options is "not set"?

Reply:
```
“xino”

Enabled with the mount option “xino=auto” or “xino=on”, with the module option “xino_auto=on” or with the kernel config option CONFIG_OVERLAY_FS_XINO_AUTO=y.
```

```
“inode index”

Enabled with the mount option or module option “index=on” or with the kernel config option CONFIG_OVERLAY_FS_INDEX=y.
```